package.json

```diff
@@ -20,18 +21,18 @@
     "@starknet-react/core": "^2.8.2",
     "colord": "^2.9.3",
     "framer-motion": "^11.2.10",
-    "get-starknet-core": "^3.2.0",
+    "get-starknet-core": "^4.0.0",
     "jotai": "^2.8.2",
     "lodash-es": "^4.17.21",
     "next": "14.2.4",
     "popmotion": "^11.0.5",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
     "starknet": "^6.11.0",
-    "starknet-react-chains-next": "npm:@starknet-react/chains@3.0.0-beta.2",
-    "starknet-react-core-next": "npm:@starknet-react/core@3.0.0-beta.2",
+    "starknet-react-chains-next": "npm:@starknet-react/chains@3.0.0-beta.3",
+    "starknet-react-core-next": "npm:@starknet-react/core@3.0.0-beta.3",
```

Comment:
Didn't they publish the official release? I thought I saw that yesterday on slack